api.md

```diff
@@ -1070,6 +1156,66 @@ to the server's timezone setting.
  multi-day calls to time_bucket. The old behavior can be reproduced by passing
  2000-01-01 as the origin parameter to time_bucket.
 
+## time_bucket_gapfill() [](time_bucket_gapfill)
+
```

Comment:
Can you put in an intro that minimum references that `time_bucket_gapfill` works similarly to `time_bucket`, but with additional functionality?

Reply:
done